README.md

```diff
@@ -114,5 +115,20 @@ Set `$GOPATH` variable to point to your Go workspace directory and add `$GOPATH/
 6. Run the server and the signer using
 
   ```sh
-  goreman start
+  docker-compose up -d
   ```
+
+7. Provision a log and a map 
+```sh
+go run $GOPATH/src/github.com/google/trillian/cmd/createtree/main.go --admin_server=localhost:8090 --pem_key_path=testdata/log-rpc-server.privkey.pem --pem_key_password="towel" --signature_algorithm=ECDSA --tree_type=LOG
+go run $GOPATH/src/github.com/google/trillian/cmd/createtree/main.go --admin_server=localhost:8090 --pem_key_path=testdata/log-rpc-server.privkey.pem --pem_key_password="towel" --signature_algorithm=ECDSA --tree_type=MAP
+```
+
+Set the `LOG_ID` and `MAP_ID` environment variables in `docker-compose.yml` with the output
+of these respective commands.
+
+7. Relaunch and observe
+- `docker-compose up -d`
+- `docker-compose logs --tail=0 --follow`
+- [https://localhost:8080/v1/users/foo@bar.com](https://localhost:8080/v1/users/foo@bar.com)
```

Comment:
The open source repo only has an unauthenticated GET.  
Different providers can choose to do different things in their environments.